src/setuptools_scm/__init__.py

```diff
@@ -86,13 +87,19 @@ def dump_version(root, version, write_to, template=None):
                 os.path.splitext(target)[1], target
             )
         )
+    try:
+        parsed_version = Version(version)
+    except Exception:
 
-    parsed_version = Version(version)
-    version_fields = parsed_version.release
-    if parsed_version.dev is not None:
-        version_fields += (f"dev{parsed_version.dev}",)
-    if parsed_version.local is not None:
-        version_fields += (parsed_version.local,)
+        log = _get_logger("setuptools_scm")
+        log.exception("failed to parse version %s", version)
+        version_fields = (0, 0, 0)
```

Comment:
Wouldn't it be best to use the version as-is -- `version_fields = (version,)`? Afterall, it is perfectly possible that this is a legacy version, and I don't think we should fallback to `0.0.0` unless there is something actually requiring us to.